### PR TITLE
Added output to docs page for Features sample.

### LINKED ISSUE #14506: The Features sample shows Tag Helpers twice instead of those and View Components
In the features sample, the output from the controller shows Tag Helpers twice. The View enumerates `Model.TagHelpers` twice instead of also enumerating `Model.ViewComponents`.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2784a724-e940-4c14-3b50-b063f78c2ee5
* Version Independent ID: 41175c3a-8cb5-856b-a7e0-931edb6394db
* Content: [Application Parts in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/mvc/advanced/app-parts?view=aspnetcore-3.0)
* Content Source: [aspnetcore/mvc/advanced/app-parts.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/mvc/advanced/app-parts.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-mvc**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

### PR DESCRIPTION
**Pull request by serpent5**
Fixes #14506.

### CODE REVIEW

File: aspnetcore/mvc/advanced/app-parts.md [hunk 1]
```diff
@@ -41,14 +38,14 @@ To include views in the assembly:
 * Add the following markup to the shared project file:
 
   ```csproj
-    <ItemGroup>
-      <EmbeddedResource Include = "Views\**\*.cshtml" />
-    </ ItemGroup >
+  <ItemGroup>
```
Comment (serpent5) [inline aspnetcore/mvc/advanced/app-parts.md]: The spacing in the content was a bit off the indentation of the whole snippet wasn't aligned to the snippet in the following bullet. That appears to be fixed when running docfx locally so I hope it's fixed in its published environment too.

File: aspnetcore/mvc/advanced/app-parts.md [hunk 2]
```diff
@@ -95,10 +92,33 @@ The `GenericController` class:
...
+    - HelloController
+    - GenericController`1
+    - GenericController`1
+Tag Helpers:
+    - PrerenderTagHelper
+    - AnchorTagHelper
+    - CacheTagHelper
+    - DistributedCacheTagHelper
+    - EnvironmentTagHelper
+    ...
```
Comment (serpent5) [inline aspnetcore/mvc/advanced/app-parts.md]: There are quite a few Tag Helpers now, so the `...` here denotes that now all of them are shown. I dunno if there's a better convention for that, so let me know if you've done something like this before.

File: aspnetcore/mvc/advanced/app-parts/sample2/AppPartsSample/Views/Features/Index.cshtml [hunk 3]
```diff
@@ -27,7 +27,7 @@
 <b>View Components:</b>
 <div class="pre-scrollable" style="max-height: 100px">
     <ul>
-        @foreach (var item in Model.TagHelpers)
+        @foreach (var item in Model.ViewComponents)
```
Comment (serpent5) [inline aspnetcore/mvc/advanced/app-parts/sample2/AppPartsSample/Views/Features/Index.cshtml]: This is where we end up with an empty list.

**Comment by Rick-Anderson** [comment]
@serpent5  nice work.

**Pull request merged**